extensions/google-cloud-platform-search/package.json

```diff
@@ -26,24 +27,25 @@
     }
   ],
   "dependencies": {
-    "@google-cloud/resource-manager": "^3.0.0",
-    "@raycast/api": "^1.27.0",
-    "node-fetch": "^3.2.0"
+    "@google-cloud/resource-manager": "^5.1.0",
+    "@raycast/api": "^1.68.1",
+    "node-fetch": "^3.3.2"
   },
   "devDependencies": {
-    "@types/node": "~16.10.0",
-    "@types/node-localstorage": "^1.3.0",
-    "@types/react": "^17.0.28",
-    "@typescript-eslint/eslint-plugin": "^5.0.0",
-    "@typescript-eslint/parser": "^5.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "react-devtools": "^4.19.2",
-    "typescript": "^4.4.3"
+    "@raycast/eslint-config": "^1.0.6",
+    "@types/node": "20.8.10",
+    "@types/node-localstorage": "^1.3.3",
+    "@types/react": "18.2.27",
+    "eslint": "^8.51.0",
+    "prettier": "^3.0.3",
+    "react-devtools": "^4.10.4",
+    "typescript": "^5.2.2"
```

Comment:
I updated the devDependencies based on the new project creation template.
https://developers.raycast.com/basics/create-your-first-extension